about.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <nav class="nav1">
+            <ul>
+                <li>
+                    <i id="facebook-icon" class="fa fa-facebook"></i>
+                </li>
+                <li>
+                    <i id="twitter-icon" class="fa fa-twitter"></i>
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav class="nav2">
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
+            <ul class="mainMenu">
+                <a href="#" class="nav-logo">Tonny's Farm</a>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
```

Comment:
- [ ] I can move from Home page to About page but I cannot move from About page to Home page. It won't let me click on any button. Please make sure you make it work as stated in the requirement.